spec.bs

```diff
@@ -6340,9 +6362,11 @@ component, and top-level auctions may specify which signals to load by the `adSl
 HTTP response header.</h3>
 
 The \`<dfn http-header><code>Ad-Auction-Additional-Bid</code></dfn>\` response header provides value
-of a string in the format of `<auction nonce>:<base64-encoding of the signed additional bid>`, which
-corresponds to a single [=additional bid=]. The response may include more than one [=additional bid=]
-by specifying multiple instances of the [:Ad-Auction-Additional-Bid:] response header.
+of a string in the format of `<auction nonce>:<seller nonce>:<base64-encoding of the signed
+additional bid>`, or the legacy format of `<auction nonce>:<base64-encoding of the signed additional
+bid>`, which corresponds to a single [=additional bid=]. The response may include more than one

```

Comment:
Small language suggestion - s/which corresponds to .../either of which corresponds to .../ - just so that there's no confusion that both the new and legacy formats correspond to a single additional bid.

Reply:
Done.